resources/CICD/Bitrise/bitrise.yml

```diff
@@ -0,0 +1,163 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- push_branch: dev
+  workflow: deploy_staging
+- push_branch: master
+  workflow: deploy_appstore
+- push_branch: "*"
+  workflow: validate
+workflows:
+  deploy_staging:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@6.0: {}
+    - cache-pull@2: {}
+    - script@1:
+        title: Do anything with Script step
+    - fastlane-match@0:
+        inputs:
+        - type: adhoc
+        - team_id: "$TEAM_ID"
+        - app_id: "$BUNDLE_ID"
+        - git_url: "$MATCH_REPO_URL"
+        - decrypt_password: "$MATCH_PASSWORD"
+    - certificate-and-profile-installer@1: {}
+    - cocoapods-install@2: {}
+    - xcode-test@2: {}
+    - set-xcode-build-number@1:
+        inputs:
+        - plist_path: "$INFO_PLIST_PATH"
+    - xcode-archive@3: {}
+    - firebase-app-distribution@0:
+        inputs:
+        - firebase_token: "$FIREBASE_CI_TOKEN"
+        - app: "$FIREBASE_APP_ID"
+    - firebase-dsym-upload@2:
+        inputs:
+        - fdu_google_services_location: "$GOOGLE_SERVICE_INFO_PLIST_PATH"
+    - cache-push@2: {}
+    envs:
+    - opts:
+        is_expand: false
+      BUNDLE_ID: com.nimblehq.example.staging
+    - opts:
+        is_expand: false
+      BITRISE_SCHEME: ExampleApp Staging
+  deploy_appstore:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@6.0: {}
+    - cache-pull@2: {}
+    - script@1:
+        title: Do anything with Script step
+    - fastlane-match@0:
+        inputs:
+        - type: appstore
+        - team_id: "$TEAM_ID"
+        - decrypt_password: "$MATCH_PASSWORD"
+        - git_url: "$MATCH_REPO_URL"
+        - app_id: "$BUNDLE_ID"
+    - certificate-and-profile-installer@1: {}
+    - cocoapods-install@2: {}
+    - xcode-test@2: {}
```

Comment:
The steps for deploy workflows should be like this:
- activate-ssh-key
- git-clone
- cache-pull
- **cocoapods-install**
- **xcode-test**
- **fastlane-match**
- **certificate-and-profile-installer**
- set-xcode-build-number
- xcode-archive
- deploy-to-itunesconnect-application-loader
- firebase-dsym-upload
- cache-push

We don't need to set up Fastlane match, certificate, and profile when the test is failed